src/main/java/nl/hsac/fitnesse/fixture/slim/HttpTest.java

```diff
@@ -37,9 +39,19 @@
     private String contentType = DEFAULT_POST_CONTENT_TYPE;
     private String lastUrl = null;
     private String lastMethod = null;
+    private boolean throwExceptionOnHttpRequestFailure = false;
+
+    public void doThrowExceptionOnHttpRequestFailure() {
```

Comment:
In general I prefer to offer a method to get the current value and one to change it. (this also allows for usage in scenarios where one first saves the current value in a symbol, changes the setting to the desired state and then resets it to the initial state, without a need to know the initial state)